mediachain/translation/simple/translator.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import unicode_literals
+from mediachain.translation.translator import Translator
+
+
+class SimpleTranslator(Translator):
+
+    @staticmethod
+    def translator_id():
+        return 'SimpleTranslator/0.1'
+
+    @staticmethod
+    def translate(parsed_metadata):
+        simple_json = parsed_metadata
+
+        # extract artist Entity
+        artist_name = simple_json['artist']
```

Comment:
I'll either rework this directly or make my own version that addresses some of this